packages/ui/components/calendar/src/LionCalendar.js

```diff
@@ -605,13 +637,22 @@ export class LionCalendar extends LocalizeMixin(LitElement) {
     day.tabindex = day.central ? '0' : '-1';
     day.ariaPressed = day.selected ? 'true' : 'false';
     day.ariaCurrent = day.today ? 'date' : undefined;
+    day.disabledInfo = '';
 
```

Comment:
If the date is disabled other then min/max, it's still good to inform the screen reader user about it.

```suggestion
if (day.disabled) {
  // TODO: turn this into a translated string
  day.disabledInfo = `This date is unavailable. Please select another date`;
}
```